php/examples/PostgreSQL.php

```diff
@@ -22,18 +22,20 @@
         // So we don't have to mess around with cursors and unbuffered queries by default.
     ]);
 
+    $conn->query("DROP TABLE IF EXISTS People");
+
     // Creating a table.
-    $sql = "CREATE TABLE People (
+    $sql = "CREATE TABLE IF NOT EXISTS People (
```

Comment:
I assume this is for race condition reasons?  It's probably good to include this anyway, but we should be using different table names for different services as well just to be extra safe.  It could get very racy otherwise.